code/modules/tgui/states/admin.dm

```diff
@@ -10,3 +10,17 @@ GLOBAL_DATUM_INIT(admin_state, /datum/ui_state/admin_state, new)
 	if(check_rights_for(user.client, R_ADMIN))
 		return UI_INTERACTIVE
 	return UI_CLOSE
+
+/**
+ * tgui state: VV_state
+ *
+ * If the user is an admin, allows interaction.
+ * If not (an admin let them *see* the panel), no touchies, only seesies. or something.
+ */
+
+GLOBAL_DATUM_INIT(VV_state, /datum/ui_state/VV_state, new)
+
+/datum/ui_state/VV_state/can_use_topic(src_object, mob/user)
+	if(check_rights_for(user.client, R_ADMIN))
```

Comment:
This should be a rights check for `R_VAREDIT`. Not `R_ADMIN`.

Reply:
yyyyeah my mind completely skipped thinking about permission stuff.

So I decided to turn this into just a check for holder, since in the old VV people with R_ADMIN could at least browse vars, but not edit and whatnot. Now permissions are checked respective to each action, except for where I missed adding a permission check, which is very probable to be at least one case.